generate_life_calendar.py

```diff
@@ -132,11 +132,11 @@ def draw_row(ctx, pos_y, birthdate, date, box_size, x_margin, darken_until_date)
 
     for i in range(NUM_COLUMNS):
         fill = (1, 1, 1)
-
-        if is_current_week(date, birthdate.month, birthdate.day):
-            fill = BIRTHDAY_COLOUR
-        elif is_current_week(date, 1, 1):
-            fill = NEWYEAR_COLOUR
+        if not classic:
```

Comment:
I like the idea of having an option to disable shading of boxes to mark birthdays and new year, I just don't think "classic" is a good word for it (the original WBW calendar doesn't have any dates on it at all). The important part is that no boxes will be shaded, so how about something like --no-box-shading or --all-blank-boxes

Reply:
I wanted to remove all changes to the WBW version with one flag, and didn't think about removing just part of the vis.

Reply:
I havent tried the changes yet, it looks like they are just removing the shading for boxes at specific dates (birthdays and new year). Is that correct? If so, then I think my first comment stands, "classic" doesn't really make sense. But if my analysis is wrong, please correct me.